README.md

```diff
@@ -154,31 +155,53 @@ Install plugins within the jenkins dashboard after installation. These help with
 
 To build logger locally in your workstation, run the following command
 1. `make -C src`
-2. Verify the test-bin is created in the `src` folder
-3. Verify the build does not fail
+1. Verify the test-bin is created in the `src` folder
+1. Verify the build does not fail
     
 
 ## Build logger in local docker container (ubuntu)
 
 To build logger in ubuntu in a docker container run the following
 1. **Terminal1**:`docker build -t logger_ubuntu:latest -f docker/ubuntu/Dockerfile`
-2. **Terminal1**:`docker run --name logger logger_ubuntu:latest`
-3. **Terminal2**: Open a new terminal and enter `docker exec -it logger sh`
-4. **Container**: In the container run `/.build.sh`
-5. **Container**: `cat input/results.txt`
-6. Verify the contents of the txt file
+1. **Terminal1**:`docker run --name logger logger_ubuntu:latest`
+1. **Terminal2**: Open a new terminal and enter `docker exec -it logger sh`
+1. **Container**: In the container run `/.build.sh`
+1. **Container**: `cat input/results.txt`
+1. Verify the contents of the txt file
 
 ## Build logger in local docker container (alpine)
 
 To build logger in alpine in a docker container run the following
-1. **Terminal1**:`docker build -t logger_alpine:latest -f docker/alpine/Dockerfile`
-2. **Terminal1**:`docker run --name logger_alpine logger_alpine:latest`
-3. **Terminal2**: Open a new terminal and enter `docker exec -it logger_alpine sh`
-4. **Container**: In the container run `/.build.sh`
-5. **Container**: `cat input/results.txt`
-6. Verify the contents of the txt file
+1. **Terminal1**: `docker build -t logger_alpine:latest -f docker/alpine/Dockerfile .`
+1. **Terminal1**: `docker run --name logger_alpine logger_alpine:latest`
+1. **Terminal2**: Open a new terminal and enter `docker exec -it logger_alpine sh`
+1. **Container**: In the container run `/.build.sh`
+1. **Container**: `cat input/results.txt`
+1. Verify the contents of the txt file
 
+## Build Logger using docker-compose
 
+### Prerequisite
+- Clean root folder of and .exe as well as input and output folders
```

Comment:
Is this saying to remove *.exe and the input and output folders?

Reply:
yes if they exist remove them out of the root folder